api/db/migrations/001_initial_schema.down.sql

```diff
@@ -10,17 +10,15 @@ DROP TRIGGER IF EXISTS update_time_documents ON documents;
 
 DROP TABLE IF EXISTS documents;
 
-DROP TABLE IF EXISTS document_states;
-
 DROP TRIGGER IF EXISTS update_time_resume_reviews ON resume_reviews;
 
 DROP TABLE IF EXISTS resume_reviews;
 
-DROP TABLE IF EXISTS resume_review_states;
+DROP TYPE IF EXISTS resume_review_state;
 
 DROP TABLE IF EXISTS user_roles;
 
-DROP TABLE IF EXISTS roles;
+DROP TYPE IF EXISTS role_type;
```

Comment:
Any reason we're using `role_type` instead of just `role` since it's already a `TYPE`?

Reply:
I didn't want it to conflict with field names like `user_roles.role`. Open to a better name

Reply:
Good point, let's keep this 👍